### ISSUE TITLE
I2C code for Auth IC 2.0 C

### ISSUE THREAD

**Issue by mayurcools** [etwmc/Personal-HomeKit-HAP#66]
sorry for sounding like a noob but is there any provision for raspberry pi auth IC 2.0C communication via I2C for raspberry pi 2/3?

**Comment by etwmc** [comment]
What do you mean?

**Comment by mayurcools** [comment]
well, for functionality in MFi mode, apple authentication co-processor (an IC) is needed and it talks to host processor over I2C. Does this library takes care of it?

**Comment by etwmc** [comment]
You have a co-processor to use? 
I mean, I can see if I have found the protocol, but I doubt most people would use it

**Comment by mayurcools** [comment]
yes, I do have a co-processor, I have made a HAT PCB for raspberry pi

**Comment by etwmc** [comment]
If I remember correctly, you need to change the Bonjour flag for MFi to 1, and add the MFi challenge prior the pair request (Basically, you got some input, you push it to the chip, and send back the result from the chip. )
Sorry can't be more helpful. To keep this project stay on github as long as possible, I don't have access to the MFi membership, so I have to remember the stuff I did 3 years ago and make sense of it.
